build-aux/app.drey.Biblioteca.Devel.json

```diff
@@ -40,7 +39,8 @@
           "type": "dir",
           "path": ".."
         }
-      ]
+      ],
+      "post-install": ["cp -a /usr/share/runtime/docs/doc/. /app/share/doc/"]
```

Comment:
I think we are missing some docs of WebKit and JavascriptCore which are present in ``/usr/share/runtime/docs/gtk-doc/html/``, we should copy them as well?